src/lib/viewers/BaseViewer.js

```diff
@@ -979,11 +982,23 @@ class BaseViewer extends EventEmitter {
 
         const options = boxAnnotations.getOptions && boxAnnotations.getOptions();
 
+        // Set disabled ftux cursor flag depending on if the local storage key is set
+        let isDocumentFtuxCursorDisabled = false;
```

Comment:
It seems like this could be inlined and converted to a constant.

Reply:
Removed this line as it is no longer needed.